intelmq/bin/intelmqctl.py

```diff
@@ -204,7 +204,10 @@ def bot_stop(self, bot_id, getstatus=True):
                 if self.__status_process(pid):
                     log_bot_error('running', bot_id)
                     return 'running'
-                self.__remove_pidfile(bot_id)
+                try:
+                    self.__remove_pidfile(bot_id)
```

Comment:
Isn't the try except something we should catch *inside* the __remove_pidfile function?